### DISCUSSION TITLE
Trying to connect MySQL between TotalEconomy-1.8.1-Sponge and TotalEconomy-0.13.2-Paper

### DISCUSSION THREAD

**Discussion by ghost** [ericgrandt/TotalEconomy#433] [Q&A] [answered]
There is an issue where TotalEconomy-0.13.2-Paper doesn't update balance on MySQL database, while TotalEconomy-1.8.1-Sponge is correctly updating the balance

**Comment by ericgrandt** [comment, marked as answer]
Hi @doarlock, TotalEconomy-1.8.1 for Sponge is actually a much older version of Total Economy (versioning is a bit weird, sorry about that) and uses a different structure for storing data. If you're using the same database for 1.8.1 Sponge as you are for 0.13.2 Paper, that'll be the issue. Currently for both Paper and Sponge, 0.13.2 is the latest version.

If you'd like to move from 1.8.1 (old Sponge version) to 0.13.2 (latest version for Sponge and Paper), you'll have to recreate the database (lose all your existing data) or manually migrate it (keep most of your data). Unfortunately, there is currently no tool to do that migration automatically. Apologies for the inconvenience.
